### PR TITLE
fix: cli error message when <snyk test> command fails.

### PR DESCRIPTION
**Pull request by anthogez**
- [x] Ready for review
- [x] Follows [CONTRIBUTING](https://github.com/snyk/snyk/blob/master/.github/CONTRIBUTING.md) rules
- [x] Reviewed by Snyk internal team

#### What does this PR do?
fix: cli error message when <snyk test> command fails.
before was appearing only `intenal error`

#### Screenshots
Screen Shot 2020-02-26 at 19 00 36

### CODE REVIEW

File: test/acceptance/cli-test/cli-test.generic.spec.ts [hunk 1]
```diff
@@ -58,7 +58,7 @@ export const GenericTests: AcceptanceTests = {
       } catch (err) {
         t.equal(
           err.userMessage,
-          'cli error message',
+          'Could not test ruby-app, \nplease check that the version and name are correct. See more details on what is supported `snyk help`',
```
Comment (gitphill) [inline test/acceptance/cli-test/cli-test.generic.spec.ts]: remove newline char?

Comment (anthogez) [reply test/acceptance/cli-test/cli-test.generic.spec.ts]: Thanks Phill

File: src/lib/errors/failed-to-get-vulns-from-unavailable-resource.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import { CustomError } from './custom-error';
+
+const errorNpmMessage =
+  'Please check the version and package name and try running `snyk test` again.\nFor additional assistance, run `snyk help` or check out our docs \n(link to: https://support.snyk.io/hc/en-us/articles/360003851277#UUID-ba99a73f-110d-1f1d-9e7a-1bad66bf0996).';
+const errorRepositoryMessage =
+  'Try testing this repository at https://snyk.io/test/.\nFor additional assistance, run `snyk help` or check out our docs \n(link to: https://support.snyk.io/hc/en-us/articles/360003851277#UUID-ba99a73f-110d-1f1d-9e7a-1bad66bf0996).';
```
Comment (gitphill) [inline src/lib/errors/failed-to-get-vulns-from-unavailable-resource.ts]: do you need 'link to:' and brackets just display URL?

Comment (anthogez) [reply src/lib/errors/failed-to-get-vulns-from-unavailable-resource.ts]: It was suggested to display it that way by content-asks.

**Pull request merged**